pkg/tsdb/azuremonitor/azure-resource-graph-datasource.go

```diff
@@ -256,13 +257,13 @@ func (e *AzureResourceGraphDatasource) unmarshalResponse(res *http.Response) (Az
 
 func getAzurePortalUrl(azureCloud string) (string, error) {
 	switch azureCloud {
-	case "AzureCloud":
+	case azidentity.AzurePublicCloud:
```

Comment:
`setting`, not `azidentity`

Reply:
I think you are looking at an outdated commit. See the latest one.

Reply:
You're right, sorry.